packages/hardhat-solhint/src/index.ts

```diff
@@ -84,7 +105,10 @@ function printReport(reports: any) {
 subtask("hardhat-solhint:run-solhint", async (_, { config }) => {
   const { processPath } = require("solhint/lib/index");
   return processPath(
-    join(config.paths.sources, "**", "*.sol").replace(/\\/g, "/"),
+    relative(
```

Comment:
Because [ignore](https://github.com/kaelzhang/node-ignore#1-pathname-should-be-a-pathrelatived-pathname) requires relative paths to properly work (used by Solhint [here](https://github.com/protofire/solhint/blob/master/lib/index.js#L65))

Reply:
Done. Let me know if it's clear enough :)

Reply:
I think this would be a bit more readable if it was done in two or three statements, with interspersed comments. Sometihng like this:

```
// we need a relative path because...
const sourcesPath = relative(config.paths.root, config.paths.sources);

// we create a glob that matches all .sol files
const solFilesGlob = join(sourcesPath, "**", "*.sol");

// globs only use forward slashes, even on windows
const normalizedGlob = solFilesGlob.replace(...)
```

Wdyt @yhuard?

Reply:
Thanks for your feedback @fvictorio I've refactored the code as you suggested.